utils/sigs/sigs.go

```diff
@@ -1,260 +1,134 @@
 package sigs
 
 import (
-	"bytes"
 	"encoding/binary"
 	"errors"
 	"fmt"
-	"strings"
 
 	btcSecp256k1 "github.com/btcsuite/btcd/btcec"
 	"github.com/cosmos/cosmos-sdk/client"
 	"github.com/cosmos/cosmos-sdk/crypto"
 	sdk "github.com/cosmos/cosmos-sdk/types"
 	"github.com/lavanet/lava/utils"
-	conflicttypes "github.com/lavanet/lava/x/conflict/types"
-	pairingtypes "github.com/lavanet/lava/x/pairing/types"
 	tendermintcrypto "github.com/tendermint/tendermint/crypto"
 	"github.com/tendermint/tendermint/crypto/secp256k1"
 )
 
```

Comment:
can we add tests to sigs

Reply:
is this enough? see https://github.com/lavanet/lava/commit/fa6bc8ebe290dce3ebda0cb260429ec0e41876fe